hyperspy/conftest.py

```diff
@@ -1,12 +1,17 @@
 # Configure mpl and traits to work in a headless system
 from traits.etsconfig.api import ETSConfig
 ETSConfig.toolkit = "null"
-import matplotlib
-matplotlib.use("Agg")
+
+# pytest-mpl 0.7 already import pyplot this line is called, so setting the 
+# matplotlib backend to 'agg' as early as we can will be useless.
+# However, resetting the rcParams to matplotlib default does the job.
+import matplotlib.pyplot as plt
+plt.rcParams.clear()
```

Comment:
Yes and this tests is actually failing using the `TkAgg` on my system... :(
The error says "AssertionError: Expected 'mock' to be called once. Called 11 times."...